Typeracer/Controllers/HomeController.cs

```diff
@@ -1,20 +1,42 @@
 using System.Diagnostics;
-using System.Net.Mime;
-using System.Runtime.InteropServices;
 using Microsoft.AspNetCore.Mvc;
 using Typeracer.Models;
 using System.Text;
+using Typeracer.Context;
+using Typeracer.Exceptions;
 
 
 namespace Typeracer.Controllers;
 
 public class HomeController : Controller
 {
     private readonly ILogger<HomeController> _logger;
+    private readonly AppDbContext _dbContext;
 
-    public HomeController(ILogger<HomeController> logger)
+    public HomeController(ILogger<HomeController> logger, AppDbContext appDbContext)
     {
         _logger = logger;
+        _dbContext = appDbContext;
+
+        List<Gamemode> simpleGamemodes = new List<Gamemode>();
+        simpleGamemodes.Add(Gamemode.Standard);
+        simpleGamemodes.Add(Gamemode.Hardcore);
+
+        List<Gamemode> shortGamemode = new List<Gamemode>();
+        shortGamemode.Add(Gamemode.Short);
+        
+        try
+        {
+            InsertParagraphFileToDb("paragraph1.txt", simpleGamemodes);
+
+            InsertParagraphFileToDb("paragraph2.txt", shortGamemode);
+            
+        }
+        catch (NoAddToDBException e)
+        {
+            Console.WriteLine(e.Message, "Intentional? - " + e.Intentional);
```

Comment:
Maybe we should change Console.Writeline to ILogger everywhere?

Reply:
yeah, sorry for not replying earlier! yes, changing it is the best option